Pipfile

```diff
@@ -1,30 +1,38 @@
 [[source]]
+
 verify_ssl = true
 name = "pypi"
 url = "https://pypi.python.org/simple"
 
+
```

Comment:
Who added these blank lines?

Reply:
Please see the answer [here](https://github.com/marius92mc/github-stargazers/pull/50#issuecomment-361176170).